### ISSUE TITLE
Faster-RNNLM

### ISSUE THREAD

**Issue by alonshmilo** [alonshmilo/MedicalData_jce#29]
# Issue: Faster-RNNLM

### Explanation:
Faster Recurrent Neural Network Language Modeling Toolkit with Noise Contrastive Estimation and Hierarchical Softmax.

### Checklist:
- [x] Code
- [x] Reading
- [x] Documenting reading
- [x] Project book
- [x] Advisor consult

**Comment by alonshmilo** [comment]
Reading and writing - myself

**Comment by alonshmilo** [comment]
Added to presentation
